packages/common/src/ceramic-api.ts

```diff
@@ -200,5 +200,5 @@ export interface MultiQuery {
   /**
    * Load a previous version of the stream based on unix timestamp
    */
-  atTime?: number
+  loadOpts?: LoadOpts
```

Comment:
loadStream just calls this `opts`, we should probably do the same here for consistency.